src/components/Pagination/PaginationRowBack.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { Icon } from '../Icon';
+
+/**
+ * PaginationRowBack component for Patternfly React
+ */
+const PaginationRowBack = ({
+  className,
+  page,
+  messagesFirstPage,
+  messagesPreviousPage,
+  onFirstPage,
+  onPreviousPage,
+  ...props
+}) => {
+  const classes = cx('pagination', 'pagination-pf-back', className);
+  return (
+    <ul className={classes} {...props}>
+      <li className={page === 1 ? 'disabled' : ''}>
+        <a
+          href="#"
+          title={messagesFirstPage}
+          onClick={e => {
+            e.preventDefault();
+            onFirstPage(e);
+          }}
+        >
+          <Icon type="fa" name="angle-double-left" />
```

Comment:
I think that the icon misses class `"i"`here and everywhere else on the buttons. Therefore the icons are a bit smaller than what they should be. See the difference:
Your PR:
![pagination_pr](https://user-images.githubusercontent.com/1186631/34986882-8d84c508-fab9-11e7-9835-1913e1b90fe3.png)
vs. patternfly definition:
![pagination_pr_i](https://user-images.githubusercontent.com/1186631/34986884-8f260412-fab9-11e7-9dc3-f2b7ac25256c.png)

Reply:
done